utils/nctl/sh/generators/wg_110.sh

```diff
@@ -46,8 +46,8 @@ done
 
 # Set defaults.
 amount=${amount:-1000000}
-gas_payment=${gas_payment:-200000}
-gas_price=${gas_price:-10}
+gas_payment=${payment:-200000}
```

Comment:
Are `$payment` and `$gas` being set in the case statement above?

Reply:
i expand the variable names from gas -> gas_price and payment -> gas_payment so I think they should be good.  So the user will enter nctl-wg-100 gas=10 payment=100000 but internally those input params are mapped to gas_price and gas_payment and defaulted accordingly.

Reply:
I see, so on the CLI, `payment=X` causes `X` to be stored in a var called `$gas_payment` in the case statement, which makes sense. However, on this line, if I'm understanding Bash correctly, we're saying "set the value of `$gas_payment` to be the value of `$payment` if `$payment` is defined, otherwise use a default value of 200000". But it seems `$payment` itself is never defined, so it'll always be the default value?

Reply:
perhaps you're right.  as boys has kicked off I will issue a patch next week if necessary.

Reply:
Sounds good to me!